packages/components/src/components/hds/breadcrumb/item.ts

```diff
@@ -27,6 +33,20 @@ export interface HdsBreadcrumbItemSignature {
 }
 
 export default class HdsBreadcrumbItem extends Component<HdsBreadcrumbItemSignature> {
+  @tracked linkToExternal: LinkTo | null = null;
+
+  constructor(owner: Owner, args: HdsBreadcrumbItemSignature['Args']) {
+    super(owner, args);
+
+    void this.setupComponent();
```

Comment:
[question] just a thought, but if we move the check on `this.args.isRouteExternal` here, some computational cycles can be saved? (_I think_) 

(even if we lose the abstraction in the check that now is shared in the helper)

Reply:
I was contemplating the same thing, and since this would be applicable to very small amunt of cases we could really save on some rerenders, I will modify this 👍🏻